src/webgl/p5.Matrix.js

```diff
@@ -82,10 +82,26 @@ p5.Matrix.prototype.set = function (inMatrix) {
   if (inMatrix instanceof p5.Matrix) {
     this.mat4 = inMatrix.mat4;
     return this;
-  }
-  else if (inMatrix instanceof GLMAT_ARRAY_TYPE) {
+  } else if (inMatrix instanceof GLMAT_ARRAY_TYPE) {
     this.mat4 = inMatrix;
     return this;
+  } else if (arguments.length === 16) {
```

Comment:
looks like this case is not indicated as an option in the docs, can we add it?

Reply:
yes. thanks for catching this and other incongruities! I can get to this tomorrow & resubmit.

Reply:
question for this one: the reason to do 16 args is to get around array creation, because this now gets used every frame for camera work and creating arrays slows down the computation—but it feels a bit much to list all 16 arguments here. I went with 

```
 * @param {Number}          [n00..n33] 16 numbers passed by value to avoid
 *                                     array copying.
```
Does that seem ok? From what I can tell, the Matrix class isn't included in the docs but other devs would want to see this for reference.

Reply:
ah I see. this makes sense to me.